lib/chitter.rb

```diff
@@ -0,0 +1,34 @@
+require 'pg'
+
+class Chitter
+
+  attr_reader :id, :message, :text
+
+  def initialize(id:, message:, text:)
+    @id  = id
+    @message = message
+    @text = text
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_manager_test')
+    else
+      connection = PG.connect(dbname: 'chitter_manager')
+    end
```

Comment:
Consider using a Database connection class here and seperate out database functionality